mappings/net/minecraft/item/ItemArmor.mapping

```diff
@@ -6,13 +6,29 @@ CLASS none/acw net/minecraft/item/ItemArmor
 		FIELD d GOLD Lnone/acw$a;
 		FIELD e DIAMOND Lnone/acw$a;
 		FIELD f name Ljava/lang/String;
-		METHOD c getResourceItem ()Lnone/aeo;
+		FIELD h protectionAmounts [I
+		FIELD i enchantability I
+		FIELD j equipSound Lnone/nk;
+		FIELD k toughness F
+		METHOD <init> (Ljava/lang/String;ILjava/lang/String;I[IILnone/nk;F)V
+			ARG 3 name
+		METHOD a getEnchantability ()I
+		METHOD b getEquipSound ()Lnone/nk;
```

Comment:
I'm 60/40 on this. `getXSound` sounds much nicer but `getSoundX` feels more proper, just like we wouldn't call blocks `DirtBlock`, but `BlockDirt`. I'd gravitate towards the latter.
( React on this comment with :+1: for `getSoundEquip`, :-1: for `getEquipSound`. )